### PR TITLE
Added scikit-learn

### PR DESCRIPTION
**Pull request by KOLANICH**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

A lots people build their libraries upon sklearn extending its functionality.

Please note that there is no logo added. The logos are in the repo, https://github.com/scikit-learn/scikit-learn/tree/master/doc/logos , the repo seems to be under BSD, but I don't want to take any legal risks about it, because they may say "the code is under bsd, but the artwork is proprietary".

### CODE REVIEW

File: topics/scikit-learn/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+aliases: sklearn
+created_by: David @cournape Cournapeau, The scikit-learn developers
```
Comment (nayafia) [inline topics/scikit-learn/index.md]: We just use names (not handles), could you edit to the following?

> David Cournapeau

File: topics/scikit-learn/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+display_name: scikit-learn
+github_url: https://github.com/scikit-learn/scikit-learn
+released: January 05, 2010
+short_description: scikit-learn is a Python module for machine learning.
+topic: scikit-learn
+related: scikit, python
+url: https://github.com/scikit-learn/scikit-learn/releases
+wikipedia_url: https://en.wikipedia.org/wiki/Scikit-learn
+---
+scikit-learn is a widely used de-facto standard Python module for classic machine learning built on top of SciPy.
```
Comment (nayafia) [inline topics/scikit-learn/index.md]: _"de-facto standard"_ seems redundant, could you remove `de-facto` here?

File: topics/scikit-learn/index.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+aliases: sklearn
+created_by: David @cournape Cournapeau, The scikit-learn developers
+display_name: scikit-learn
+github_url: https://github.com/scikit-learn/scikit-learn
+released: January 05, 2010
+short_description: scikit-learn is a Python module for machine learning.
+topic: scikit-learn
+related: scikit, python
+url: https://github.com/scikit-learn/scikit-learn/releases
```
Comment (nayafia) [inline topics/scikit-learn/index.md]: this seems to be the correct URL: http://scikit-learn.org/

**Comment by nayafia** [comment]
@KOLANICH Thanks for this! Left some comments in review.

> Please note that there is no logo added. The logos are in the repo, https://github.com/scikit-learn/scikit-learn/tree/master/doc/logos , the repo seems to be under BSD, but I don't want to take any legal risks about it, because they may say "the code is under bsd, but the artwork is proprietary".

The logo is used on their Wikipedia page (https://en.wikipedia.org/wiki/Scikit-learn) so I'm guessing it's fine to use. If you feel comfortable uploading a version of it as 288x288 image, that'd be great. If not, that's ok too!

**Comment by KOLANICH** [comment]
1 On Wikipedia page it is marked as copyrighted
2 See https://github.com/scikit-learn/scikit-learn/issues/10436

**Comment by nayafia** [comment]
@KOLANICH no worries! If you could address the comments in review we can merge.

**Comment by MikeMcQuaid** [comment]
@KOLANICH Gentle ping on the above comments, thanks.

File: topics/scikit-learn/index.md (hunk 3 above)
Comment (KOLANICH) [reply topics/scikit-learn/index.md]: It's http. We shouldn't encourage http without TLS.

File: topics/scikit-learn/index.md (hunk 2 above)
Comment (KOLANICH) [reply topics/scikit-learn/index.md]: It's not standardized, it's just very popular. See https://en.wikipedia.org/wiki/De_facto_standard for more info.

File: topics/scikit-learn/index.md (hunk 1 above)
Comment (KOLANICH) [reply topics/scikit-learn/index.md]: Sorry, no. Could you do that for me?

Comment (MikeMcQuaid) [reply topics/scikit-learn/index.md]: @KOLANICH Ping on this.

File: topics/scikit-learn/index.md (hunk 2 above)
Comment (MikeMcQuaid) [reply topics/scikit-learn/index.md]: @KOLANICH Please make the change as requested, thanks!

File: topics/scikit-learn/index.md (hunk 3 above)
Comment (MikeMcQuaid) [reply topics/scikit-learn/index.md]: @KOLANICH Regardless, please change it, thanks.

File: topics/scikit-learn/index.md (hunk 2 above)
Comment (nayafia) [reply topics/scikit-learn/index.md]: can remove `de-facto standard` and just say "widely used", that should be sufficient.

File: topics/scikit-learn/index.md (hunk 1 above)
Comment (KOLANICH) [reply topics/scikit-learn/index.md]: Fixed.

File: topics/scikit-learn/index.md (hunk 3 above)
Comment (MikeMcQuaid) [reply topics/scikit-learn/index.md]: @KOLANICH Still missing this change, thanks.

File: topics/scikit-learn/index.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+wikipedia_url: https://en.wikipedia.org/wiki/Scikit-learn
+---
+scikit-learn is a widely used [de facto standard](https://en.wikipedia.org/wiki/De_facto_standard) Python module for classic machine learning built on top of SciPy.
```
Comment (MikeMcQuaid) [inline topics/scikit-learn/index.md]: Please either use the unlinked word "standard" or just remove "de facto standard" entirely. We're aware of what a de facto standard is, we just don't agree it's the best wording. Thanks!

File: topics/scikit-learn/index.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+aliases: sklearn
+created_by: David Cournapeau, The scikit-learn developers
+display_name: scikit-learn
+github_url: https://github.com/scikit-learn/scikit-learn
+released: January 05, 2010
+short_description: scikit-learn is a Python module for machine learning.
+topic: scikit-learn
+related: scikit, python
+url: https://scikit-learn.org/
```
Comment (MikeMcQuaid) [inline topics/scikit-learn/index.md]: This is not a valid https page. Again, please make it link to http://scikit-learn.org/. Thanks!

Comment (KOLANICH) [reply topics/scikit-learn/index.md]: I know. Shame on them that their HTTPS doesn't work.

**Comment by MikeMcQuaid** [comment]
@KOLANICH Any chance you'd consider reopening this? It's a good PR that it would be a shame to lose just over a non-working HTTPS page. Thanks ❤️!

**Comment by KOLANICH** [comment]
No, I'm not going to promote using non-encrypted non-authenticated insecure protocols in 2018. If you wanna promote it, feel free to take the files from my PR, modify them as you like and post your own PR attributing it to yourself. But I'm not going to be a part of that PR.

**Comment by MikeMcQuaid** [comment]
@KOLANICH Ok, disappointing but up to you.

**Comment by nayafia** [comment]
Opened https://github.com/github/explore/pull/162.

**Pull request closed without merging**